docs/pages/image.js

```diff
@@ -9,53 +9,52 @@ import docgen, { type DocGen } from '../components/docgen.js';
   return (
-    <Page title="Image">
-      <PageHeader name="Image" description={generatedDocGen?.description} />
+    <Page title={generatedDocGen?.displayName}>
+      <PageHeader name={generatedDocGen?.displayName} description={generatedDocGen?.description} />
 
       <GeneratedPropTable generatedDocGen={generatedDocGen} />
 
       <Card
         description={`
-    One thing that might be unusual is that the \`width\` and the \`height\` of the
-    component are required, yet the image will scale to the size of its container.
-    This is so that the placeholder's size can be calculated before the image has
-    rendered.
-
-    While the exact dimensions supplied aren't used (only the ratio between them is
-    considered), you should always try to try to supply the exact dimensions of the
-    source image requested.
+One thing that might be unusual is that the \`width\` and the \`height\` of the component are required, yet the image will scale to the size of its container.
+
+This is so that the placeholder's size can be calculated before the image has rendered.
+
+While the exact dimensions supplied aren't used (only the ratio between them is considered), you should always try to try to supply the exact dimensions of the source image requested.
   `}
         name="Dimensions"
       />
 
       <Example
         id="placeholders"
         description={`
-    The color you pass into \`Image\` will be used to fill the placeholder that shows up
-    as an image loads. The example shown has an empty \`src\` prop provided so it remains
-    a placeholder.
+The color you pass into Image will be used to fill the placeholder that shows up as an image loads. The example shown contains an image with a transparent background which allows you to visualize the placeholder color. When Mask is used to create rounded corners on Image, the placeholder color may leak through on the corners. To prevent this, remove the placeholder color."
   `}
         name="Placeholders"
         defaultCode={`
-<Column span={6}>
-  <Image
-    alt="example.com"
-    color="rgb(111, 91, 77)"
-    naturalHeight={564}
-    naturalWidth={564}
-    src=""
-  />
-</Column>
+<Flex>
+{
+  ["rgb(111, 91, 77)", "black", "var(--color-background-shopping)"].map((color) => (
+    <Column span={2}>
+      <Image
+        alt="example.com"
+        color={color}
+        naturalHeight={1}
+        naturalWidth={1}
+        src="https://d3cy9zhslanhfa.cloudfront.net/media/BBEEEEC7-E954-4223-B5A061E37D0C03E2/CE43CF95-DE36-465B-956EFB21C9CC9C04/webimage-0311D236-89DC-4404-9D9B1452C865159C.png"
+      />
+    </Column>
+  ))
+}
+</Flex>
 `}
       />
 
       <Example
-        description="
-    You can overlay content on an Image by passing it children.
-  "
+        description="You can overlay content on an Image by passing it children."
         name="Overlay"
         defaultCode={`
-<Box height={500} paddingX={2} width={250}>
+<Box height={(751/564)*200} width={200}>
```

Comment:
This is confusing. If this is usage we're recommending, we should provide documentation for it — otherwise let's just use 266 directly.

Reply:
It's math... but sure!

Reply:
DONE